### ISSUE TITLE
Spotify playlists

### ISSUE THREAD

**Issue by alexanderdoe** [586837r/node-red-contrib-alexa-remote2#186]
Is it possible to play playlists from spotify? Alexa keeps telling me that she can't find the playlist

**Comment by quadhammer** [comment]
Yes, it is, but I'm not sure how that refers to this node, unless you were trying to use a custom command to start it.  

You just say, Alexa, listen to the chill playlist.

(Where chill is the name of your playlist)

There's also a spotify node
